src/torchcodec/decoders/_core/VideoDecoder.cpp

```diff
@@ -1587,7 +1620,8 @@ double VideoDecoder::getMaxSeconds(const StreamMetadata& streamMetadata) {
 // VALIDATION UTILS
 // --------------------------------------------------------------------------
 
-void VideoDecoder::validateActiveStream() {
+void VideoDecoder::validateActiveStream(
+    std::optional<AVMediaType> avMediaType) {
```

Comment:
Is there ever a time when we will want to generically check the valid active stream but not know if we're audio or video? Is there any place we're doing it currently? If the answer to the second question is no, I'd prefer to make this a required field now. We can revisit when we encounter that scenario.

Reply:
Yes I should have commented on this: I made this optional because one the decoding PR (still WIP), there are calls to this function where we want to acept both audio and video streams (namely in getFramesPlayedInRange())